resources/views/layouts/navigation.blade.php

```diff
@@ -16,8 +16,8 @@
                 </form>
                 <!-- Navigation Links -->
                 <div class="hidden space-x-8 sm:-my-px sm:ml-10 sm:flex">
-                    <x-nav-link :href="route('dashboard')" :active="request()->routeIs('dashboard')">
-                        {{ __('Dashboard') }}
+                    <x-nav-link :href="route('account-profile')" :active="request()->routeIs('account-profile')">
```

Comment:
can we move this link to the dropdown, and only show if the user is logged in?

Reply:
or, it looks like you added it to the dropdown actually, so i think you can remove this link

Reply:
I actually changed this to route to '/' rather than the 'account-profile' so the user is able to click a "Home" button and return to the index page.

I can delete if you think that's unnecessary, however.